tests/test_browsers.py

```diff
@@ -2,106 +2,233 @@
 from pathlib import Path
 
 from .context import browser_history
-from .utils import become_linux, become_windows, become_mac, change_homedir # pylint: disable=unused-import
+from .utils import (
+    become_linux,
+    become_windows,
+    become_mac,
+    change_homedir,
+)  # pylint: disable=unused-import
 
 # pylint: disable=redefined-outer-name,unused-argument
 
+
 def test_firefox_linux(become_linux, change_homedir):
     """Test history is correct on Firefox for Linux"""
     f = browser_history.browsers.Firefox()
-    output = f.fetch()
-    his = output.get()
+    output = f.fetch_history()
+    his = output.histories
     assert len(his) == 5
-    assert his[0] == (datetime.datetime(2020, 8, 3, 0, 29, 4,
-                                      tzinfo=datetime.timezone(datetime.timedelta(seconds=19800),
-                                                               'IST')),
-                    'https://www.mozilla.org/en-US/privacy/firefox/')
-    profs = f.profiles()
+    assert his[0] == (
+        datetime.datetime(
+            2020,
+            8,
+            3,
+            0,
+            29,
+            4,
+            tzinfo=datetime.timezone(datetime.timedelta(seconds=19800), "IST"),
+        ),
```

Comment:
This linting seems excessive.

Reply:
I agree. This formatting was done by black ([1](https://github.com/psf/black/), [2](http://www.locallyoptimal.com/blog/2019/08/23/why-you-should-use-black-for-your-python-style-linting/)) and so we didn't have much control over what it did.

I tried to fix it manually but black seems to not like it :laughing:

Reply:
I guess black knows best then 😂😂😂